hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/Invocation_OnMemberLeftTest.java

```diff
@@ -88,6 +102,43 @@ public void whenMemberLeaves() throws Exception {
         }
     }
 
+    private static final AtomicBoolean MASTER_EXCEPTION_HANDLED = new AtomicBoolean();
```

Comment:
a static field in a parallel test is bound to create problems with test methods executing concurrently, best to create one as local variable in the test method and pass it as argument to `FailingUnresponsiveMasterOperation`

Reply:
the issue is that ```FailingUnresponsiveMasterOperation``` is ```DataSerializable```. And I want to have the same instance of ```AtomicBoolean``` after ser/deser

That's why I used static field. My thoughts on that:
- I can do comment on the field that it cannot be reuse on other tests - If we have static field that is used in just one test it should break anything
- I can mark test as serial for sanity
Or maybe you know some better solution to such problem?

Reply:
let's just add a comment on the field, thanks!